### PR TITLE
Show policy description annotation in UI

### PR DESCRIPTION
**Pull request by willkutler**
https://issues.redhat.com/browse/ACM-3189

### CODE REVIEW

File: frontend/src/routes/Governance/common/util.tsx [hunk 1]
```diff
@@ -537,3 +550,19 @@ export function handlePolicyAutomationSubmit(
...
+
+export function formatDescriptionForDropdown(desc: string) {
+  const formattedDescription = []
+  if (desc) {
+    const descLines = desc.split(/\r?\n/)
+    for (let i = 0; i < descLines.length; i++) {
+      const line = descLines[i]
+      formattedDescription.push(<span>{line}</span>)
+      if (i < descLines.length - 1) {
+        formattedDescription.push(<br></br>)
```
Comment (mprahl) [inline frontend/src/routes/Governance/common/util.tsx:562]:
```suggestion
        formattedDescription.push(<br>)
```

File: frontend/src/routes/Governance/common/util.tsx [hunk 2]
```diff
@@ -537,3 +550,19 @@ export function handlePolicyAutomationSubmit(
...
   })
 }
+
+export function formatDescriptionForDropdown(desc: string) {
+  const formattedDescription = []
+  if (desc) {
+    const descLines = desc.split(/\r?\n/)
+    for (let i = 0; i < descLines.length; i++) {
+      const line = descLines[i]
+      formattedDescription.push(<span>{line}</span>)
```
Comment (mprahl) [inline frontend/src/routes/Governance/common/util.tsx:560]: Would this be better as a paragraph tag?

File: frontend/src/routes/Governance/common/util.tsx (hunk 1 above)
Comment (mprahl) [reply frontend/src/routes/Governance/common/util.tsx:562]: If we do the paragraph tag then you probably don't need these break HTML elements.

File: frontend/src/routes/Governance/policies/CreatePolicy.test.tsx [hunk 3]
```diff
@@ -62,8 +62,9 @@ describe('Create Policy Page', () => {
 
     await new Promise((resolve) => setTimeout(resolve, 500))
 
-    // step 1 -- name and namespace
+    // step 1 -- name, description, and namespace
     userEvent.type(screen.getByRole('textbox', { name: 'Name' }), 'policy1')
+    userEvent.type(screen.getByRole('textbox', { name: 'Description' }), 'Test policy description')
```
[the remark this answers is not in the document]
Comment (willkutler) [reply frontend/src/routes/Governance/policies/CreatePolicy.test.tsx:67]: As far as I know there are no unit tests that verify that - there's tests for setting the fields and that changes in the form are reflected in the sync editor, but making that into a unit test would probably require mocking the entire create policy page, so I think it is better done as an e2e/QE test.

**Pull request merged**